services/QuillLMS/spec/lib/rule_feedback_history_spec.rb

```diff
@@ -107,7 +107,7 @@ def rule_factory(&hash_block)
         # feedback_histories
         f_h1 = create(:feedback_history, rule_uid: so_rule1.uid, entry: "f_h1 lorem")
         f_h2 = create(:feedback_history, rule_uid: so_rule1.uid, entry: "f_h2 ipsum")
-
+        
```

Comment:
Accidental new whitespace?

Reply:
I added this line to make the groupings more individually consistent.